columnflow/tasks/reduction.py

```diff
@@ -130,7 +132,7 @@ def run(self):
         for (events, sel, *diffs), pos in self.iter_chunked_io(
             input_paths,
             source_type=["coffea_root"] + (len(input_paths) - 1) * ["awkward_parquet"],
-            read_options=[{"iteritems_options": {"filter_name": load_columns_nano}}] + (len(input_paths) - 1) * [None],
+            read_columns=[read_columns, None] + (len(input_paths) - 2) * [read_columns],
```

Comment:
It wasn't possible before with plain ak/parquet since one needs to peek into the file to check which masks exist in the first place. However, this is now possible with dask_awkward which only loads meta data by default. Added in d32f807e :)